src/Tribe/JSON_LD/Event.php

```diff
@@ -87,4 +90,29 @@ public function get_data( $posts = null, $args = array() ) {
 		return $return;
 	}
 
+	/**
+	 * Normalizes the price entry to make it compatible with JSON-LD guidelines.
+	 *
+	 * @param string|int $price
+	 *
+	 * @return string
+	 */
+	protected function normalize_price( $price ) {
+		$map = array(
+			'/^\\s*free\\s*$/i' => '0',
+		);
+
+		foreach ( $map as $normalization_regex => $normalized_price ) {
+			if ( preg_match( $normalization_regex, '' . $price ) ) {
+				$price = $normalized_price;
+			}
+		}
+
+		// normalize the decimal separator
+		$price = str_replace( ',', '.', $price );
```

Comment:
Yes, there is and I had not considered that.
I'm thinking about pulling the decimal separator from WordPress but that's not an option and it really does not tell much.
Is there a way to get WordPress to tell me the decimal and thousands separator?